packages/database/src/migrations/20200812043136-AddTongaHpuSettingTypeMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+import { insertObject } from '../utilities/migration';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const COLOUR_MAP_OVERLAY = {
+  id: 'TO_HPU_NCD_Risk_Factory_Screening_Type_Setting_Unique',
+  name: 'NCD Risk Factor Screenings by Setting Type',
+  userGroup: 'Tonga Health Promotion Unit',
+  dataElementCode: 'value',
+  isDataRegional: false,
+  measureBuilder: 'selectUniqueValueFromEventsPerOrgUnit',
+  measureBuilderConfig: {
+    valueToSelect: 'HP31n',
+    dataValues: {
+      HP31n: '*',
+    },
+    programCode: 'HP02',
+    entityAggregation: {
+      aggregationType: 'REPLACE_ORG_UNIT_WITH_ORG_GROUP',
+      dataSourceEntityType: 'village',
+      aggregationEntityType: 'facility',
+    },
+  },
+  presentationOptions: {
+    values: [
+      {
+        icon: 'circle',
+        name: 'Church',
+        color: 'darkblue',
+        value: 'Church',
+      },
+      {
+        icon: 'circle',
+        name: 'School',
+        color: 'orange',
+        value: 'School',
+      },
+      {
+        icon: 'circle',
+        name: 'Workplace',
+        color: 'pink',
+        value: 'Workplace',
+      },
+      {
+        icon: 'circle',
+        name: 'Community',
+        color: 'lightblue',
+        value: 'Community',
+      },
+      {
+        icon: 'circle',
+        name: 'Multiple Settings',
+        color: 'yellow',
+        value: 'NO_UNIQUE_VALUE',
+      },
+    ],
+    hideFromPopup: true,
+    measureLevel: 'Facility',
+  },
+  countryCodes: '{"TO"}',
+  projectCodes: '{"fanafana"}',
+};
+
+const TOTAL_MAP_OVERLAY = {
+  id: 'TO_HPU_NCD_Risk_Factory_Screening_Type_Setting_Type_ZZZ_Total',
```

Comment:
This is a bit of hack lol. The order the linked measures get sorted is alphabetical by id, and since I wanted the `Total` to be last in the list I add this `ZZZ_` in front to force that... Too hacky? 😝

Reply:
Seems like an fairly acceptable one as it is a config 'hack' rather than a code hack.